sprint/sprint-guide.md

```diff
@@ -0,0 +1,50 @@
+(sprint_guide)=
+# Sprint Guide
+
+Here are some guidelines and suggestions about the sprint for participants and volunteers!
+
+## General Information
+
+- Be Polite
+- Ask questions
+- Don't use any discriminatory terms
+- Use gender neutral pronouns unless you know someone's preferred pronouns
+- Don't use words like "guys", instead say "folks" or "people"
+- Be inclusive and kind
+
+The sprint will be happening on Zoom, the sign up instructions are available on Meetup, please try to join on the starting time so you can recieve proper help and guidance for the sprint. There will be breakout rooms inside the Zoom call that participants will be assigned to:
+- Women and Non binary folks: this breakout room is specifically for women and non binary folks, we hope you have a meaningful space here where you're heard!
```

Comment:
I have mixed feelings about the wording. Maybe it's just me but I can see this being interpreted as women and non binary folks will only be heard here